scrapers/BelAmi.yml

```diff
@@ -0,0 +1,30 @@
+name: "BelAmi"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - belamionline.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="ContentPlaceHolder1_LabelTitle"]
+      Details:
+        selector: //*[@id="ContentPlaceHolder1_LabelDescription"]|//div[@class='bottom']/p[2]
+        concat: " "
```

Comment:
Not sure why the union was needed for the first part
For scenes with multiple paragraphs (`https://newtour.belamionline.com/playvideo.aspx?VideoID=15308`) only a partial description is scraped
```yaml
      Details:
        selector: //p[span[@id="ContentPlaceHolder1_LabelDescription"]]/following-sibling::p
        concat: "\n\n"
```
seems to work ok

Reply:
Confimed and resolved.